### PR TITLE
Add pickup notification emails

### PR DESCRIPTION
**Pull request by nicksellen**
Ok, basic functionality is there, todo:

- [x] check it manually a bit more
- [x] add tests

### CODE REVIEW

File: foodsaving/groups/templates/group_summary.mjml [hunk 1]
```diff
@@ -59,7 +59,7 @@
                     <mj-divider/>
                     <mj-text font-size="13px" line-height="18px" color="#777">
                         <p>
-                            {% trans %}You are receiving this because you are subscribed to <strong>Weekly summary</strong> notifications for this group.{% endtrans %}
+                            {% trans %}You are receiving this because you are subscribed to <strong>Weekly summary</strong> emails for this group.{% endtrans %}
```
Comment (tiltec) [inline foodsaving/groups/templates/group_summary.mjml]: I like "e-mails" instead of "email" in written text

Comment (nicksellen) [reply foodsaving/groups/templates/group_summary.mjml]: http://grammarist.com/style/e-mail-email/ says:

> A few editorially conservative publications still prefer e-mail to email, but most of the English-speaking world has adopted the unhyphenated form. In a Google News search covering 2011 and the start of 2012, there are approximately six instances of email for every e-mail, a dramatic shift from a couple of years ago. And the unhyphenated form is even more common outside newswriting.

I get the impression the hypenated version is used in German more?

Comment (tiltec) [reply foodsaving/groups/templates/group_summary.mjml]: Ah yes, that might be the reason... well, onto more consistency in future.

File: foodsaving/groups/serializers.py [hunk 2]
```diff
@@ -274,7 +274,10 @@ def update(self, instance, validated_data):
 
 class GroupMembershipAddNotificationTypeSerializer(serializers.Serializer):
     notification_type = serializers.ChoiceField(
-        choices=(GroupNotificationType.WEEKLY_SUMMARY,),
+        choices=(
```
Comment (tiltec) [inline foodsaving/groups/serializers.py]: Ah, much easier than expected. I guess that also works for model fields: "An iterable (e.g., a list or tuple) consisting itself": https://docs.djangoproject.com/en/2.0/ref/models/fields/

**Pull request merged**